src/webhook_launcher/app/models.py

```diff
@@ -190,6 +190,18 @@ def matches(self, proj_name):
                 return True
         return False
 
```

Comment:
PEP8 nitpicks as this module has been cleaned up.

- Spaces around ` = `  in `found=None`
- Line too long `raise Multiple...`
- 2 blank lines expected before following class definition